src/app/modules/shared/services/logging/sentry.service.ts

```diff
@@ -8,12 +8,26 @@ import { LoggedInUser } from '../../../login/models/logged-in-user.model';
 import { LoggingService } from './logging.service';
 import { LogLevel } from './log-level.model';
 import { FileLoggingService } from './file-logging.service';
+import { AppVersion } from 'src/app/core/models/app-version.model';
 
 @Injectable({
   providedIn: 'root'
 })
 export class SentryService implements LoggingService {
-  constructor(private appVersionService: AppVersionService, private fileLoggingService: FileLoggingService) {}
+
+  private versionInfo: AppVersion;
+
+  constructor(
+    appVersionService: AppVersionService,
+    private fileLoggingService: FileLoggingService
+  ) {
+    this.versionInfo = appVersionService.getAppVersion();
+    this.log(
```

Comment:
Hvis du fjerner 
`if (this.fileLoggingService.isReady())`
i linje 90 i denne klassen, får vi logget versjonsnummeret til fil. Det er ganske mange logginger vi går glipp av i fila under oppstarten pga. denne if-setninga. Jeg testet å ta den bort, og da funket det. FileLoggingService køer opp loggemeldinger til den er klar til å logge til fil, så det går bra om man kaller log() litt tidlig.

Reply:
Så bra - bare fiks dette hvis du vil og ta den inn etterpå.

Reply:
Ok. Har gjort det nå.